data-raw/ogden2005/predictors.csv

```diff
@@ -1,6 +1,6 @@
 value,j_day,pred,pred_subcategory
 20,NA,host_den,deer
-200,NA,host_den,rodent
+200,NA,host_den,mouse
```

Comment:
there's now a requirement that this (`mouse`) matches with the name of a vector parameter in a transition